### PR TITLE
[14_0_X] Custom nano for e/gamma

### PR DESCRIPTION
**Pull request by swagata87**
Backport of https://github.com/cms-sw/cmssw/pull/42607, as requested in https://github.com/cms-sw/cmssw/pull/42607#issuecomment-1966358677. Seems urgent from this comment by @RSalvatico in original PR: 

> There is also a chance to test the custom nano production this week if we manage to integrate it,

### CODE REVIEW

**Comment by swagata87** [comment]
type egamma

**Comment by swagata87** [comment]
enable nano

**Comment by swagata87** [comment]
please test

File: Configuration/PyReleaseValidation/python/relval_nano.py [hunk 1]
```diff
@@ -239,12 +248,15 @@ def subnext(self):
 workflows[_wfn()] = ['muDPGNANO130Xrun3', ['ZMuSkim2023DRAWRECO13.0', 'muDPGNANO_data13.0']]
 workflows[_wfn()] = ['muDPGNANOBkg130Xrun3', ['ZeroBias2023DRAW13.0', 'muDPGNANOBkg_data13.0']]
 workflows[_wfn()] = ['muPOGNANO_data13.0', ['MuonEG2023MINIAOD13.0', 'muPOGNANO_data13.0']]
+workflows[_wfn()] = ['EGMNANOdata130Xrun3', ['MuonEG2023MINIAOD13.0', 'EGMNano_data13.0']]
+workflows[_wfn()] = ['EGMNANOmc130X', ['TTBarMINIAOD13.0', 'EGMNano_mc13.0']]
```
Comment (vlimant) [inline Configuration/PyReleaseValidation/python/relval_nano.py:252]: please move this up to line 245 below `NANOmc130X`

**Comment by vlimant** [comment]
please test

**Comment by vlimant** [comment]
+1

**Comment by mandrenguyen** [comment]
backport of #42607

**Comment by mandrenguyen** [comment]
+1

**Comment by subirsarkar** [comment]
+Upgrade

**Comment by AdrianoDee** [comment]
+pdmv

**Comment by antoniovilela** [comment]
+1

**Pull request merged**